lib/HTTP/BrowserDetect.pm

```diff
@@ -1820,7 +1827,8 @@ sub _init_version {
         # specific approaches and go straight to the generic ones.
     }
     elsif ( $browser_tests->{edge} ) {
-        ( $major, $minor, $beta ) = $ua =~ m{Edge/(\d+)(.\d+)(.\d+)?};
+        ( $major, $minor, $beta ) = $ua =~ m{Edge/(\d+)\.(\d+)\.?(\d+)?}i;
+        ( $major, $minor, $beta ) = $ua =~ m{(?:Edg|EdgA|EdgiOS)/(\d+)\.(\d+)\.?(\d+)\.?(\d+)?}i unless defined $major;
```

Comment:
EdgeHTML engine version is captured instead because of not exposed Edge browser version.

> only the EdgeHTML engine version number is exposed
https://blogs.windows.com/msedgedev/2017/10/05/microsoft-edge-ios-android-developer/#jDBWCiCz8f9Lg2gy.97

Another suggestion is:
```suggestion
        ( $major, $minor, $beta ) = $ua =~ m{(?:Edg|EdgA|EdgiOS)/(\d+)\.(\d+)\.?(\d+)\.?(\d+)?}i;
```
The browser version will be guessed and maybe EdgeHTML version can be got.